.github/workflows/notify.yml

```diff
@@ -18,6 +18,8 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Fetch the full history instead of a shallow clone
```

Comment:
Curious why we need this here

Reply:
Can see why it's needed in release.yml since it deals with changesets and tags, but not sure about this workflow